standards/.pylintrc

```diff
@@ -135,7 +135,9 @@ enable-variables=yes
 # Tells wether we should check for unused import in __init__ files.
 init-import=no
 
-
+# A regular expression matching the name of dummy variables (i.e. expectedly
+# not used).
+dummy-variables-rgx=_$|dummy
```

Comment:
are we allowing only _ and dummy for dummy variable or any thing start with dummy prefix?